deployments/salt/templates/agent_config.yaml

```diff
@@ -31,7 +29,7 @@ exporters:
     verbosity: normal
 
 service:
-  extensions: [memory_ballast]
+  extensions:
```

Comment:
Not sure if leaving this empty is valid. It doesn't hurt to add `zpages` instead at least